Dockerfile

```diff
@@ -10,7 +10,7 @@ WORKDIR /usr/src/app
 COPY . /usr/src/app
 
 # Install dependencies (pip or conda)
-#RUN pip install -r requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
I introduce dataclasses into the requirements.txt
It is included in later python as native modules but the python in the docker image